lib/main.js

```diff
@@ -319,6 +319,69 @@ if (options.interactiveTests) {
     })
   })
 
+  ava.test('supervisor: Supervisor update test through the API', async (test) => {
+    return utils.runManualTestCase(test, {
+      prepare: [
+        'Ensure the device is online and running an application',
+        'Logging into the device (ssh to the host OS)'
+      ],
+      do: [
+        'Copy and paste the below script into the shell to run it - it will take the device\'s credentials, calls the ' +
+        'API for the supervisor for cross-check, update the API entry for this device, and calls the supervisor updater ' +
+        'to check if it finds the updated info:\n\n' +
+        'source /etc/resin-supervisor/supervisor.conf\n' +
```

Comment:
Pasting the whole script here looks terrible. Can we put the code into version control in another repo (ie the supervisor, so we ensure the supervisor team keeps it updated), and provide a link?